app/views/notification_mailer/digest.html.erb

```diff
@@ -65,17 +65,21 @@
             <td colspan="12" align="center">
               <table cellspacing="0" cellpadding="0">
                 <tr>
+                  <td width="20px">&nbsp;</td>
                   <td width="590px">
                     <hr style="border-top: 0.5px solid #ddd; border-bottom: 0.5px solid #ddd; border-right: 0.5px solid transparent; border-left: 0.5px solid transparent;">
                   </td>
+                  <td width="20px">&nbsp;</td>
                 </tr>
                 <tr>
                   <td height="10px"></td>
```

Comment:
no `&nbsp;`? I'm assuming this is email client related, but it caught my eye it's missing here and also in L85, is it because we're using `height` and not `width` so it isn't needed?

Reply:
Sounds like a @MattBudz question. If you can throw in :D

Reply:
>is it because we're using height and not width so it isn't needed?

That's right. These cells are used to create vertical white space so we don't need `&nbsp;`.

`&nbsp;` is used to show horizontal white space correctly in FF.